### ISSUE TITLE
Bug: calcite-action-bar:  we dont see a scrollbar anymore when the window size is small

### ISSUE THREAD

**Issue by kevindoshier** [Esri/calcite-components#3025]
### Actual Behavior

When the window size is shrunk vertically we no longer see the scrollbar

### Expected Behavior

We should see a scrollbar so we can access the other actions
![scrollbar](https://user-images.githubusercontent.com/11093161/133150518-852a2239-14ba-49e0-b3cb-7195bdf15eb9.png)

### Reproduction Steps or Sample

1.

### Relevant Info

_Version_: `@esri/calcite-components@<version>`

- [ ] CDN
- [ ] NPM package

**Comment by driskull** [comment]
@jcfranco @julio8a can we triage this for this sprint?

**Comment by jcfranco** [comment]
Added to current sprint! I assigned it to you in the meantime. cc @benelan

**Comment by driskull** [comment]
@jcfranco @kevindoshier I'm unable to locally reproduce this so I think it might be a timing issue. However, I have some improvements that may fix it. PR coming.

**Comment by zaramatheson** [comment]
@kevindoshier @driskull Is it expected that this should be in the MV today? We are still seeing an issue on 1280x1024 monitor and smaller window sizes

![image](https://user-images.githubusercontent.com/50747646/133675803-8498eb95-7f19-47fb-95a4-1316a037ba75.png)

**Comment by benelan** [comment]
@zaramatheson it should be in beta.65, is that the version being used?

**Comment by kevindoshier** [comment]
Todays build is using beta.65,  also Im able to repro here if I shrink the window really small

https://esri.github.io/calcite-components/?path=/story/components-shell--basic

**Comment by driskull** [comment]
@kevindoshier there's not enough icons to reproduce it in that sample.

**Comment by kevindoshier** [comment]
@driskull - thats true, will this work?
https://codepen.io/kevindoshier/pen/rNwpKVe

**Comment by driskull** [comment]
Seems like its working correctly there if you set the height to 400px in the code instead of 800.

@kevindoshier Can you tell me why this shell container has a height of 0? seems like that might be an issue.

![image](https://user-images.githubusercontent.com/1231455/133679068-058a284e-8067-4981-b8f5-006f04ce58c4.png)

Also, seems like the body of the page should have a margin and padding of 0 set.

**Comment by kevindoshier** [comment]
I set it to 400 and still dont see a scrollbar on the actionBar

![image](https://user-images.githubusercontent.com/11093161/133679327-80c5f1ee-c658-4221-80b0-6e12dbd75baa.png)


Im he height of 0 coming from the mapViewer?

**Comment by driskull** [comment]
> Is the height of 0 coming from the mapViewer?

Yes.

**Comment by kevindoshier** [comment]
Im not sure why that would be 0 in the mapViewer

**Comment by driskull** [comment]
The body of the MV should set the margin/padding to 0 and a height/width of 100% I think.

**Comment by kevindoshier** [comment]
Sounds good Ill make those changes 👍

**Comment by driskull** [comment]
Seems like its still an issue. I think I might know what the problem is but we should probably pick it up next release.

@kevindoshier we can try a MV workaround to trigger a resize on the action-bar elements possibly. 

Can you try setting an initial height on the action-bar elements like `height: 0px;` and then remove that style once the page is loaded? Maybe via a class or something?

**Comment by kevindoshier** [comment]
Yeah Ill experiment with that, but I agree I think lets fix this at 9.4, its already like this on production too

**Comment by kevindoshier** [comment]
I tested the scenario of setting the height on the calcite-action-bar to 0 using a class, then once the page loads removing it, but it doesnt seem to change anything, the overflow menus still do not appear

**Comment by driskull** [comment]
Ok, I think we need to pick it up in next release then.

**Comment by kevindoshier** [comment]
@driskull - this looks good!  Thank you
